### PR TITLE
Bbwp 50

### PR DESCRIPTION
**Pull request by gerardhros**
# BBWPC v1.2.1 2023-09-06
## Changed
* remove function `bbwp_check_lsw` and internal package table `lsw`, #BBWP-50
* add argument `B_LSW_ID` to functions `bbwp` and `bbwp_field_indicators`
* remove arguments `a_lon` and `a_lat` from functions `bbwp` and `bbwp_field_indicators`

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Package: BBWPC
 Type: Package
 Title: Calculator for BedrijfsBodemWaterPlan (BBWP)
-Version: 1.1.1
+Version: 1.2.1
```
Comment (BrentWHR) [inline DESCRIPTION:4]: I think there are breaking changes, then version should be boosted to 2.0.0

Comment (gerardhros) [reply DESCRIPTION:4]: agreed. will update to 2.0.0

**Review by SvenVw — APPROVED** [review]
LGTM

File: R/bbwp_main.R [hunk 2]
```diff
@@ -81,12 +78,35 @@ bbwp <- function(B_SOILTYPE_AGR, B_LU_BBWP,B_GWL_CLASS, B_SC_WENR, B_HELP_WENR,B
...
+              "B_P_WA_SD","B_P_SG_SD","B_FE_OX_SD","B_AL_OX_SD","B_SA_W_SD","B_RO_R_SD")
+    
+    # check LSW format and column names
+    checkmate::assert_data_table(LSW,nrow = length(A_SOM_LOI))
+    checkmate::assert_subset(colnames(LSW),choices = cols)
+    
+    # check if all B_LSW_ID are in the LSW data.table
+    checkmate::assert_subset(LSW$B_LSW_ID,choices = B_LSW_ID)
+    
+  }
```
Comment (BrentWHR) [inline R/bbwp_main.R:108]: Should a warning be raised to inform the user that a default LSW is used and that they may want to supply their own LSW data?

Comment (gerardhros) [reply R/bbwp_main.R]: thanks. warning has been added

**Pull request merged**